broker/lib/controllers/ServiceBrokerApiController.js

```diff
@@ -191,19 +450,55 @@ class ServiceBrokerApiController extends FabrikBaseController {
       .catch(ServiceBindingAlreadyExists, conflict);
   }
 
+  putBindingV2(req, res) {
+    const params = _(req.body)
+      .omit('plan_id', 'service_id')
+      .set('binding_id', req.params.binding_id)
+      .value();
+
+    function done(credentials) {
+      res.status(CONST.HTTP_STATUS_CODE.CREATED).send({
+        credentials: credentials
+      });
+    }
+
+    function conflict(err) {
+      /* jshint unused:false */
+      res.status(CONST.HTTP_STATUS_CODE.CONFLICT).send({});
+    }
+
+    // Check if write locked
+    return lockManager.isWriteLocked(req.params.instance_id)
```

Comment:
Should we check `if (req.manager.name === CONST.INSTANCE_TYPE.DIRECTOR)`, it will save a call to apiserver.